migrations/000031_cpu_usage_event_types_unique.up.sql

```diff
@@ -0,0 +1,123 @@
+BEGIN;
+
+SET search_path = public, pg_catalog;
+
+-- Update the events to reference the original event type with the corresponding name.
+-- Fixes issues where an event references an event type that was added to the 
+-- database multiple times.
+UPDATE ONLY cpu_usage_events 
+    SET event_type_id = (
+        SELECT id FROM cpu_usage_event_types 
+        WHERE name = 'cpu.hours.add' 
+        ORDER BY last_modified ASC 
+        LIMIT 1
+    ) 
+    WHERE event_type_id in (
+        SELECT id FROM cpu_usage_event_types
+        WHERE name = 'cpu.hours.add'
+        AND last_modified != (
+            SELECT last_modified FROM cpu_usage_event_types
+            WHERE name = 'cpu.hours.add'
+            ORDER BY last_modified ASC
+            LIMIT 1
+        )
```

Comment:
There's not really a need to change this, but I think that you might be able to get away with omitting this condition, just because replacing the event type ID with the same ID won't do any harm.